src/components/constants.js

```diff
@@ -0,0 +1 @@
+export const THE_BLUE = "#337AB7";
```

Comment:
Is this how you want to do non-file scoped JS constants? In Ivory this is how we do constants defined/used in file vs. `theBlue` for exported constants.

Reply:
Ha, no, that's just another late-hour refactor. Changing it to match the conventions, thanks!